docs/learning/WriteABridge.md

```diff
@@ -1,108 +1,154 @@
-# Write an iModel Bridge
+## How to write an iTwin Connector
+Note: iTwin Connectors were also known as iModel Bridges.  Though the term "iModel Bridge" has been deprecated in favor of "iTwin Connector", for legacy and backward compatibility reasons, you may find the older terminology used in this documentation.  You can consider them the same.
 
-As explained in the [overview](../learning/imodel-bridges.md), a "bridge" is a program that:
+### Connecting data to an iTwin
 
-1. Reads information from a data source,
-2. Aligns the source data with the BIS schema and preferably a domain schema, and
-3. Writes BIS data to an iModel.
+![iTwin Connector Steps](./imodel_connector_steps.png)
 
-Specifically, a bridge must:
+There are three main steps that a connector needs to undertake to bring data into a digital twin
 
-* [Open a local briefcase copy](./backend/IModelDb.md) of the iModel that is to be updated.
-* Import or Update Schema
-  * Possibly [import an appropriate BIS schema into the briefcase](./backend/SchemasAndElementsInTypeScript.md#importing-the-schema)  or upgrade an existing schema.
-  * [Push](./backend/IModelDbReadwrite.md#pushing-changes-to-imodelhub) the results to the iModelServer.
-* Convert Changed Data
-  * Connect to the data source.
-  * Detect changes to the source data.
-  * [Transform](../learning/imodel-bridges.md#data-alignment) the new or changed source data into the target BIS schema.
-  * Write the resulting BIS data to the local briefcase.
-  * Remove BIS data corresponding to deleted source data.
-  * Obtain required [Locks and Codes](./backend/ConcurrencyControl.md) from the iModel server and/or code server.
-  * [Push](./backend/IModelDbReadwrite.md#pushing-changes-to-imodelhub) changes to the iModel server.
+- Extract data from the input source
+- Transform and align the data to the digital twin.
+- Generate [changesets](https://github.com/imodeljs/imodeljs/tree/master/docs/learning/IModelHub/index.md#the-timeline-of-changes-to-an-imodel) and load data into an iModel.

```

Comment:
the link for changesets can `./IModelHub/index.md#the-timeline-of-changes-to-an-imodel`.  Then it will be a relative link instead of going to the source code.

Reply:
Same comment about a lot of the links to other documents in this article.

Reply:
just pushed - thanks